### PR TITLE
Add to docs Migration plan examples

### PR DESCRIPTION
**Pull request by goosemania**
closes #5849
https://pulp.plan.io/issues/5849

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,8 +50,68 @@ $ git clone https://github.com/pulp/pulp-2to3-migration.git
...
+  "plugins": [
+    {
+      "type": "iso",
+      "repositories": [
+        {
+          "name": "file",
+          "repository_versions": [
+            {
+              "pulp2_repository_id": "file",
+              "distributor_ids": [
```
Comment (dralley) [inline README.md]: Update this, ```pulp2_distributor_repository_ids```

**Comment by dralley** [comment]
Also this should close issue https://pulp.plan.io/issues/5849

File: README.md (hunk 1 above)
Comment (goosemania) [reply README.md]: Thanks!

**Comment by dralley** [comment]
@goosemania Needs a changelog entry

**Pull request merged**